postinstall.js

```diff
@@ -166,7 +181,7 @@ function findMainPath(installDirectory) {
 	return directory;
 }
 
-function replaceMain(mainAppPath) {
+/*function replaceMain(mainAppPath) {
```

Comment:
This replaces the iOS app's `main.m` with an automatic integration. This does not work for us because we have many integrations that require native setup (ie firebase, remote config) in our `AppDelegate.mm`.

Reference: https://developer.goacoustic.com/acoustic-campaign/docs/add-the-ios-sdk-to-your-app#integrate-the-sdk-with-your-app

Reply:
So I've commented this out so it doesn't happen when the plugin gets installed.